ue/ueReceiverModule.m

```diff
@@ -282,6 +282,21 @@
 			obj.Bits.ok = tot - diff;
 		end
 			
+		% This is used when a UE has not been scheduled in a specific round for the metrics recorded
+		function obj = logNotScheduled(obj)
+			obj.Blocks = struct('tot',0,'err',0,'ok',0);
+      obj.Bits = struct('tot',0,'err',0,'ok',0);
+		end
+
+		% This is used when a UE received waveform cannot be demodulated
+		function obj = logNotDemodulated(obj)
+			obj.PostEvm = 100;
```

Comment:
Maybe EVM should be NaN? I mean, if we cannot demodulate, EVM cannot be computed.